yaml/_error.ts

```diff
@@ -6,15 +6,8 @@
 import type { Mark } from "./_mark.ts";
 
 export class YamlError extends Error {
-  constructor(
-    message = "(unknown reason)",
-    protected mark: Mark | string = "",
-  ) {
+  constructor(message: string, mark?: Mark) {
     super(`${message} ${mark}`);
```

Comment:
When mark is not passed, now the error message becomes something like `YamlError: message undefined` instead of `YamlError: abc`